agents/ovm/fence_ovm.py

```diff
@@ -0,0 +1,193 @@
+					logging.debug("Job succeed, but '%s' power status is %s", options["--plug"], status)
+				else:
+					return True
+		time.sleep(int(options["--stonith-status-sleep"]))
+		if int(options["--power-timeout"]) > 0 and _ >= int(options["--power-timeout"]):
+			logging.error("Job failed: Timed out waiting for %s to power %s", options["--plug"], options["--action"])
+			return False
+
+def get_outlet_list(connection, options):
+	virtual_machines = send_command(connection, options, "/Vm")
+	result = {}
+	for vm in virtual_machines:
+		status = state.get(vm["vmRunState"])
+		result[vm["id"]["value"]] = (vm["name"], status)
+	return result
+
+def connect(options):
+	connection = requests.Session()
+	password = options.get("--password")
+	if "--username" in options:
+		connection.auth = (options["--username"], password)
+	else:
+		base_uri = "https://{}:{}/ovm/core/wsapi/rest".format(options["--ip"], options["--ipport"])
+		connection.mount(base_uri, SSLAdapter(options["--ssl-client-certificate-file"], password))
+	connection.verify = ("--ssl-secure" in options) or ("--ssl-insecure" not in options)
+	if connection.verify:
+		if "--ssl-insecure" in options:
+			logging.warning("The option '--ssl-insecure' is ignored")
+	else:
+		import urllib3
+		urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+	connection.headers.update({
+		"Accept" : "application/json",
+		"Content-Type" : "application/json"
+	})
+	return connection
+
+def send_command(connection, options, resources_path, method="GET"):
+	uri = "https://{}:{}/ovm/core/wsapi/rest{}".format(options["--ip"], options["--ipport"], resources_path)
+	timeout = int(options["--shell-timeout"])
+	timeout = timeout if timeout != 0 else None
+	try:
+		if method == "GET":
+			response = connection.get(uri, timeout=timeout)
+		elif method == "PUT":
+			response = connection.put(uri, timeout=timeout)
+		else:
+			logging.error("The method '%s' is not supported yet", method)
+			fail(EC_GENERIC_ERROR)
+	except ConnectionError as e:
+		logging.error(str(e))
+		fail(EC_CONNECTION_LOST)
+	except Timeout as e:
+		logging.error(str(e))
+		fail(EC_TIMED_OUT)
+	except RequestException as e:
+		logging.error(str(e))
+		fail(EC_GENERIC_ERROR)
+	try:
+		result = response.json()
+	except JSONDecodeError as e:
+		logging.error("%s is not valid JSON. Status code is %d", repr(str(response.content, response.encoding or "utf-8")), response.status_code)
+		fail(EC_GENERIC_ERROR)
+	logging.debug(result)
+	if (method == "GET" and response.status_code != 200) or (method == "PUT" and response.status_code != 201):
+		message = result.get("message") if type(result) is dict else None
+		if message is None:
+			logging.error("Failed: Remote returned %d for '%s' request to %s", response.status_code, method, uri)
+		else:
+			logging.error("Failed: %s: %d: %s", method, response.status_code, result["message"])
+		fail(EC_GENERIC_ERROR)
+	return result
+
+def define_new_opts():
+	all_opt["ssl_client_cert_file"] = {
```

Comment:
You should update this to "ssl_client_certificate_file", as it's used for stdin parsing of parameters (e.g. from Pacemaker), so using the same name here will make it easier for users to create the resource based on the parameters they used in CLI testing.

Reply:
I  understand. After modification, I will create a new PR.

Reply:
You can just update this PR. no need to create a new PR each time.

Reply:
The modification is done.
Could you review this update in this PR ?
I'm concerned that this update is not reflected in this PR.

Reply:
Since this was already merged you'll have to create a new PR. Fabio probably didnt notice it was merged.